neighborhoods_backend/settings.py

```diff
@@ -23,7 +23,7 @@
 SECRET_KEY = os.environ.get('DJANGO_SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = os.environ.get('DEBUG') == "True"
+DEBUG = os.environ.get('DEBUG') == "False"
```

Comment:
This is not what you want.

before the change:
If DEBUG is 'True' -> True
if DEBUG is 'False' -> `False`
if DEBUG is unset, or anything else -> `False`

After:
if DEBUG is "False", ("False" == "False" evaluates to `True`, so) Debug is `True`
if DEBUG is "True" ("True" == "False") -> `False`
anything else: `False`

I (personally) think a more clear approach would be:
`DEBUG = bool(os.environ.get('DEBUG', False))`
which takes the environment val DEBUG (or returns False if it's not set) and then casts it into a boolean value.

so 'True' -> `True`, and 'False' -> `False`.

Reply:
damn, yes that makes a lot more sense.  I'll update.